packages/react/.changeset/chilled-drinks-mate.md

```diff
@@ -0,0 +1,5 @@
+---
+"@gadgetinc/react": patch
+---
+
+Updated PolarisAutoSubmit to expose Polaris button props
```

Comment:
Can you add an example here so we know what to update the docs with?

Reply:
Updated to show an example of Polaris button props being used in the button